server/src/main/java/org/elasticsearch/repositories/blobstore/FileRestoreContext.java

```diff
@@ -188,12 +188,12 @@ private void afterRestore(SnapshotFiles snapshotFiles, Store store, StoreFileMet
 
         /// now, go over and clean files that are in the store, but were not in the snapshot
         try {
+            store.deleteQuiet("restore");
```

Comment:
I couldn't find any usage of a "restore" file and this seems to be dead code to me, but maybe I'm missing something?

Reply:
Looks right to me, the deletion of this file was added way back in 59d9f7e1579 but it's not obvious why.

Reply:
Looks like dead code to me too, so +1 on removing